lib/codegen/build.rs

```diff
@@ -18,12 +18,19 @@
 // The build script expects to be run from the directory where this build.rs file lives. The
 // current directory is used to find the sources.
 
+extern crate rustc_version;
 extern crate cranelift_codegen_meta as meta;
 
 use std::env;
 use std::process;
+use rustc_version::{version, Version};
 
 fn main() {
+    // Detect Rust 1.26.
+    if version().unwrap() >= Version::parse("1.26.0").unwrap() {
+        println!("cargo:rustc-cfg=rust_1_26");
```

Comment:
Could you rename the cfg option to `greater_than_rust_1_26` or `after_rust_1_26`?

Reply:
Done